README.md

```diff
@@ -389,8 +390,30 @@ const firstInstallTime = DeviceInfo.getFirstInstallTime();
 // Android: 1517681764528
 ```
 
+**Notes**
```

Comment:
Can you review (update if needed) and move this section under `getFontScale()`? Seems like my first coffee didn't completely kick in when I was resolving conflict.